scripts/deployment/deploy_loanToken.py

```diff
@@ -198,6 +200,102 @@ def testDeployment(acct, sovryn, loanTokenAddress, underlyingToken, collateralTo
     print("position size is ", collateral)
     tx = sovryn.closeWithSwap(loanId, acct, collateral, True, b'')
 
+'''
+test the affiliate margin trade by entering and closing a trade position
```

Comment:
why are tests here?
can integration testing be moved to hardhat?

Reply:
There is initialized loanToken test inside the deployment script. So i tried to reuse it.